src/api/contracts.ts

```diff
@@ -25,27 +26,28 @@ const contractNames = [
 ]
 
 // breaks in rinkeby, cancel for now
-// if (process.env.NODE_ENV === 'development') {
-//   contractNames.push(
-//     'TokenOMG',               // TODO: > 0.9.0 will be deleted - use TokenERC20
-//     'TokenRDN',               // TODO: > 0.9.0 will be deleted - use TokenERC20)
-//   )
-// }
+if (process.env.FE_CONDITIONAL_ENV === 'development') {
+  contractNames.push(
+    'TokenOMG',               // TODO: > 0.9.0 will be deleted - use TokenERC20
+    'TokenRDN',               // TODO: > 0.9.0 will be deleted - use TokenERC20)
+  )
+}
 
 // fill contractsMap from here if available
 const filename2ContractNameMap = {
   EtherToken: 'TokenETH',
 }
 
 interface ContractsMap {
-  DutchExchange:  DXAuction,
-  TokenMGN:       MGNInterface,
-  TokenETH?:      ETHInterface,
-  TokenGNO?:      GNOInterface,
-  TokenOWL?:      OWLInterface,
-  TokenOMG?:      GNOInterface,
-  TokenRDN?:      GNOInterface,
+  DutchExchange:        DXAuction,
+  DutchExchangeHelper:  any,
```

Comment:
You can do here `Pick<DXAuction, functions | you | need>`

Reply:
:+1:  TS master